public/js/bundle.js

```diff
@@ -1,10 +1,10 @@
 (function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
```

Comment:
The bundle.js file should not be checked into git or GitHub. It's a big, ugly mess of a file that Heroku will generate for us upon deployment, so there's no need for it to be tracked here. That's why we put it in .gitignore at the beginning of app development.

Delete it from your GitHub repo, and be sure to delete any modifications to bundle.js that git tracks.